agent/connect_ca_endpoint_test.go

```diff
@@ -290,6 +290,4 @@ func TestConnectCARoots_PEMEncoding(t *testing.T) {
 	require.NoError(t, err)
 	pool := x509.NewCertPool()
 	require.True(t, pool.AppendCertsFromPEM(data))
-	// expecting the root cert from dc1 and an intermediate in dc2
-	require.Len(t, pool.Subjects(), 2)
```

Comment:
do we not care/ want for this check anymore 🤔 ?

Reply:
Originally this seemed like a pretty low value assertion like we were just testing the certPool add function, but I see now we're trying to validate the endpoint is returning the right number of cert. I found another way to do this.